File: Dolphin/ControlGroup/Code/Dev/ProdSparkService/src/main/java/com/mnrclara/spark/core/model/wmscorev2/InhouseTransferLine.java
package com.mnrclara.spark.core.model.wmscorev2;

import lombok.Data;

import java.sql.Timestamp;

@Data
public class InhouseTransferLine {

    private String languageId;
    private String companyCodeId;
    private String plantId;
    private String warehouseId;
    private String transferNumber;
    private String sourceItemCode;
    private Long sourceStockTypeId;
    private String sourceStorageBin;
    private String targetItemCode;
    private Long targetStockTypeId;
    private String targetStorageBin;
    private Double transferOrderQty;
    private Double transferConfirmedQty;
    private String transferUom;
    private String palletCode;
    private String caseCode;
    private String packBarcodes;
    private Long specialStockIndicatorId;
    private Long statusId;
    private String barcodeId;
    private String referenceField1;
    private String referenceField2;
    private String referenceField3;
    private String referenceField4;
    private String referenceField5;
    private String referenceField6;
    private String referenceField7;
    private String referenceField8;
    private String referenceField9;
    private String referenceField10;
    private Long deletionIndicator;
    private String remarks;
    private String createdBy;
    private Timestamp createdOn;
    private String confirmedBy;
    private Timestamp confirmedOn;
    private String updatedBy;
    private Timestamp updatedOn;
    private String companyDescription;
    private String plantDescription;
    private String warehouseDescription;
    private String sourceStockTypeDescription;
    private String targetStockTypeDescription;
    private String manufacturerName;

}

File: MobileApp/Almailem/app/src/main/java/com/tvhht/myapplication/transfers/model/InhouseTransferLine.kt
package com.tvhht.myapplication.transfers.model

import com.google.gson.annotations.SerializedName


data class InhouseTransferLine (

  @SerializedName("caseCode"                ) var caseCode                : String? = null,
  @SerializedName("companyCodeId"           ) var companyCodeId           : String? = null,
  @SerializedName("confirmedBy"             ) var confirmedBy             : String? = null,
  @SerializedName("confirmedOn"             ) var confirmedOn             : String? = null,
  @SerializedName("createdOn"               ) var createdOn               : String? = null,
  @SerializedName("createdby"               ) var createdby               : String? = null,
  @SerializedName("languageId"              ) var languageId              : String? = null,
  @SerializedName("packBarcodes"            ) var packBarcodes            : String? = null,
  @SerializedName("palletCode"              ) var palletCode              : String? = null,
  @SerializedName("plantId"                 ) var plantId                 : String? = null,
  @SerializedName("sourceItemCode"          ) var sourceItemCode          : String? = null,
  @SerializedName("sourceStockTypeId"       ) var sourceStockTypeId       : Int?    = null,
  @SerializedName("sourceStorageBin"        ) var sourceStorageBin        : String? = null,
  @SerializedName("specialStockIndicatorId" ) var specialStockIndicatorId : Int?    = null,
  @SerializedName("targetStockTypeId"       ) var targetStockTypeId       : Int?    = null,
  @SerializedName("targetItemCode"          ) var targetItemCode          : String? = null,
  @SerializedName("targetStorageBin"        ) var targetStorageBin        : String? = null,
  @SerializedName("transferConfirmedQty"    ) var transferConfirmedQty    : Int?    = null,
  @SerializedName("transferOrderQty"        ) var transferOrderQty        : Int?    = null,
  @SerializedName("transferUom"             ) var transferUom             : String? = null,
  @SerializedName("warehouseId"             ) var warehouseId             : String? = null,
  @SerializedName("manufacturerName"        ) var manufacturerName        : String? = null
)